### ISSUE TITLE
Quote all columns in generated inserts, updates

### ISSUE THREAD

**Issue by naturalethic** [porsager/postgres#57]
I use camelCase for table and column names.  These need to be quoted.  Currently when I do something like:

```
    await sql`
      INSERT INTO "Address" ${sql(address)}
    `
```

The fields aren't being quoted.

**Comment by porsager** [comment]
That's a really good point. It's a breaking change, but also wrong if someone relied on that functionality, so let's include the fix in 2.0 which should be right around the corner.

**Comment by naturalethic** [comment]
You need a hand?
